spec/features/handovers_feature_spec.rb

```diff
@@ -71,4 +70,19 @@
       end
     end
   end
+
+  describe 'COM allocation overdue handovers page' do
+    it 'works' do
```

Comment:
A better description might be _Expected output_

Reply:
Expected output is..... the COM allocation overdue handovers page?